velox/functions/sparksql/specialforms/SparkCastHooks.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/expression/CastHooks.h"
+
+namespace facebook::velox::functions::sparksql {
+
+/// This class provides spark cast hooks to allow customized behaviors of
+/// SparkCastExpr. The implementations of hooks are following Spark semantics.
+class SparkCastHooks : public exec::CastHooks {
+ public:
+  SparkCastHooks(const core::QueryConfig& config) : CastHooks(config) {}
+
+  void castTimestampToString(
```

Comment:
Would be nice to document these methods.

Reply:
Updated. Thanks.